src/app/qgsmapsavedialog.cpp

```diff
@@ -459,3 +460,9 @@ void QgsMapSaveDialog::accepted()
     }
   }
 }
+
+void QgsMapSaveDialog::showHelp()
+{
+  // description needs to be written and i'm not sure of the place yet
```

Comment:
Removed. Will mention these information in the calc file that lists user manual links and buttons, for tracking.